azurerm/internal/services/netapp/netapp_volume_resource.go

```diff
@@ -194,6 +195,41 @@ func resourceNetAppVolume() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
+
+			"data_protection_replication": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"endpoint_type": {
+							Type:     schema.TypeString,
+							Required: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								"dst",
```

Comment:
could we use the enum constants in the SDK here?

Reply:
I would leave as is since the sdk provides "src" as well which is automatically included on primary volumes and we're hidding that operation on primaries since it is all automatically and using the constants will pass validation for "src".

Reply:
can we we still not use the enu. constant for dst by its self?
```suggestion
								string(sdk.dstvalue),
```